setup.py

```diff
@@ -89,5 +89,6 @@ def walker(base, *paths):
     extras_require={
         'mysql': ['mysql-python'],
         'postgres': ['psycopg2'],
+        'proxy': ['pycurl'],
```

Comment:
It's optional and does not get installed when you install tornado from PyPI.
But this option will not work without it (as SimpleAsyncHTTPClient does not support proxying).